### PR TITLE
docs: consolidated, simplified and corrected various docs.

### PR DESCRIPTION
**Pull request by steverfrancis**
Signed-off-by: Steve Francis <steve.francis@talos-systems.com>

# Pull Request

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [ ] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [ ] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

File: website/content/v1.3/learn-more/philosophy.md [hunk 1]
```diff
@@ -52,7 +51,7 @@ Talos has always been designed with security in mind.
 With its immutability, its minimalism, its signing, and its componenture, we are
 able to simply bypass huge classes of vulnerabilities.
 Moreover, because of the way we have designed Talos, we are able to take
-advantage of a number of additional settings, such as the recommendations of the Kernel Self Protection Project (kspp) and the complete disablement of dynamic modules.
+advantage of a number of additional settings, such as the recommendations of the Kernel Self Protection Project (kspp) and completely disabling dynamic modules.
```
Comment (frezbo) [inline website/content/v1.3/learn-more/philosophy.md:54]: dynamic modules is not true anymore in the sense that custom modules can be loaded via machineconfig

File: website/content/v1.3/talos-guides/network/vip.md [hunk 2]
```diff
@@ -5,30 +5,24 @@ aliases:
...
-The IP address is real, and it is assigned to an interface.
-Instead, what actually happens is that the controlplane machines vie for
-control of the shared IP address using etcd elections.
-There can be only one owner of the IP address at any given time, but if that
+The most common approaches - reverse proxy, load
+balancer, BGP, and DNS - all require external resources, and add complexity in setting up Kubernetes.
+
+To simplify cluster creation, Talos Linux supports a "Virtual" IP (VIP) address to access the Kubernetes API server, providing high availability with no other resources required.
+
+What happens is that the controlplane machines vie for control of the shared IP address using etcd elections.
```
Comment (frezbo) [inline website/content/v1.3/talos-guides/network/vip.md:15]: vie?

Comment (TimJones) [reply website/content/v1.3/talos-guides/network/vip.md:15]: Vie sort of means "compete" or "try to win".

Comment (frezbo) [reply website/content/v1.3/talos-guides/network/vip.md:15]: > Vie sort of means "compete" or "try to win".

TIL :+1:

File: website/content/v1.3/learn-more/philosophy.md (hunk 1 above)
Comment (steverfrancis) [reply website/content/v1.3/learn-more/philosophy.md:54]: But they still are not dynamic, in the sense of you could run `insmod` and dynamically load a module while the system runs, yes?

File: website/content/v1.3/introduction/getting-started.md [hunk 3]
```diff
@@ -80,23 +78,20 @@ Thus, the format of the endpoint may be something like:
...
+The Kubernetes API Server endpoint, in order to be highly availabile, should be configured in a way that functions off all available control plane nodes. 
 There are three common ways to do this:
 
 ### Dedicated Load-balancer
 
 If you are using a cloud provider or have your own load-balancer available (such
 as HAProxy, nginx reverse proxy, or an F5 load-balancer), using
 a dedicated load balancer is a natural choice.
-Create an appropriate frontend matching the endpoint, and point the backends at each of the addresses of the Talos controlplane nodes.
+Create an appropriate frontend matching the endpoint, and point the backends at each of the addresses of the Talos controlplane nodes. (Note that given we have not yet created the control plane nodes, the IP addresses of the backends may not be known yet. We can bind hte backends to the frontend at a later point. At this stage, we just need the IP and Port of the frontend to be created, for use later on.)
```
Comment (smira) [inline website/content/v1.3/introduction/getting-started.md:89]:
```suggestion
Create an appropriate frontend matching the endpoint, and point the backends at each of the addresses of the Talos controlplane nodes. (Note that given we have not yet created the control plane nodes, the IP addresses of the backends may not be known yet. We can bind the backends to the frontend at a later point. At this stage, we just need the IP and Port of the frontend to be created, for use later on.)
```

File: website/content/v1.3/introduction/getting-started.md [hunk 4]
```diff
@@ -142,50 +137,31 @@ https://kube.cluster1.mydomain.com:6443
...
-
-Whichever way you wish to access the Talos API, be sure to note the IP(s) or
-hostname(s) so that you can configure your `talosctl` tool's `endpoints` below.
+If your control plane nodes are not directly reachable from your workstation where you run `talosctl`, then configure a load balancer for TCP port 50000 to be forwarded to the controlplane nodes. (Do not use Talos Linux's built in VIP support for accessing the Talos API, as it will not function in the event of an etcd failure, and you will not be able to access the Talos API to fix things.)
 
-**NOTE**: The [Virtual IP]({{< relref "../talos-guides/network/vip.md" >}}) method is not recommended when accessing the Talos API as it requires etcd to be bootstrapped and functional.
-This can make debugging any issues via the Talos API more difficult as issues with Talos configuration may result in etcd not achieving quorum, and therefore the Virtual IP not being available.
-In this case setting the endpoints to the IP or hostnames of the control plane nodes themselves is preferred.
+If you create a load balanccer to forward the Talos API calls, make a note of the IP or
```
Comment (smira) [inline website/content/v1.3/introduction/getting-started.md:146]:
```suggestion
If you create a load balancer to forward the Talos API calls, make a note of the IP or
```

File: website/content/v1.3/learn-more/philosophy.md [hunk 5]
```diff
@@ -6,10 +6,9 @@ description: "Learn about the philosophy behind the need for Talos Linux."
 
 ## Distributed
 
-Talos is intended to be operated in a distributed manner.
-That is, it is built for a high-availability dataplane _first_.
+Talos is intended to be operated in a distributed manner: it is built for a high-availability dataplane _first_.
 Its `etcd` cluster is built in an ad-hoc manner, with each appointed node joining on its own directive (with proper security validations enforced, of course).
-Like as kubernetes itself, workloads are intended to be distributed across any number of compute nodes.
+Like kubernetes, workloads are intended to be distributed across any number of compute nodes.
```
Comment (smira) [inline website/content/v1.3/learn-more/philosophy.md:11]:
```suggestion
Like Kubernetes, workloads are intended to be distributed across any number of compute nodes.
```

File: website/content/v1.3/learn-more/philosophy.md [hunk 6]
```diff
@@ -62,12 +61,51 @@ enforced.
...
 
 This is achievable because Talos is tightly focused to do one thing: run
-kubernetes, in the easiest, most secure, most reliable way it can.
+Kubernetes, in the easiest, most secure, most reliable way it can.
+/Users/stevefrancis/workspace/talos/website/content/v1.3/learn-more
+## Not based on X distro
+
+Talos Linux _isn't_ based on any other distribution.
+We think of ourselves as being the second-generation of
+container-optimised operating systems, where things like CoreOS, Flatcar, and Rancher represent the first generation (but the technology is not dervied from any of theose.)
```
Comment (smira) [inline website/content/v1.3/learn-more/philosophy.md:78]:
```suggestion
container-optimised operating systems, where things like CoreOS, Flatcar, and Rancher represent the first generation (but the technology is not dervied from any of those.)
```

these/those?

**Review by smira — APPROVED** [review]

**Pull request merged**